docs/design.md

```diff
@@ -55,6 +58,7 @@ They currently include:
 
 A profile expresses a desired configuration for a Kubernetes cluster.
 It is composed of
+
 - An optional Cluster Specification. If specified, it allows the profile
```

Comment:
Same as the above comment

Reply:
Same https://github.com/arlonproj/arlon/issues/311